### PR TITLE
feat(export): writes path deprection log only when LOG_CONSUL_DEPRECATION=true

### PR DESCRIPTION
**Pull request by dholmes**
Notion: https://www.notion.so/9a463a3f03ed44f9b85408d30e5c88a9

### CODE REVIEW

File: dev/export-consul.ctmpl [hunk 1]
```diff
@@ -2,6 +2,9 @@
 set -e
 
 function log_deprecated(){
+    if [ "${LOG_CONSUL_DEPRECATION}" != "true" ]; then
+        return 1
```
Comment (mloberg) [inline dev/export-consul.ctmpl:6]: with `set -e`, this is considered an error and will exit

Comment (dholmes) [reply dev/export-consul.ctmpl:6]: Why did my brain tell me to use a 1?! :doh: fixing to return 0 which seems to be what I want. Thanks.

Comment (mloberg) [reply dev/export-consul.ctmpl:6]: You can also just use `return`

**Review by mloberg — COMMENTED** [review]
It looks like some tests might be failing because we are turning logging off? Can we either update those tests or comment them out until we set this as the default again?

Also be sure to update the cache number in the install script

**Pull request merged**